color/p5js/index.md

```diff
@@ -0,0 +1,13 @@
+---  
+layout: p5js
+author: [Filip Czekajlo]
+author-url: [-]
+title: [colm]
+tags: [P5.js,[Color], [-], ]
+permalink:   gestalten-in-code/color/p5js/colm.js  
+summary: [-]
+---  
+
+
+
+![]( gestalten-in-code/color/p5js/out.png )  
```

Comment:
use relativ paths for images. The code from your index.js will be added to the page. So there is no need for the image.  

A relativ path would be  

     ![](./out.png)